README.md

```diff
@@ -62,12 +62,13 @@ topostats process
 ```
 
 If you have your own YAML configuration file (see [Usage : Configuring
-TopoStats](https://afm-spm.github.io/TopoStats/main/usage.html#configuring_topostats)) then invoke `topostats process`
-and use the argument for `--config <config_file>.yaml` that points to your file.
+TopoStats](https://afm-spm.github.io/TopoStats/main/usage.html#configuring_topostats)) then invoke `topostats`
+and use the argument for `--config <config_file>.yaml` that points to your file with an associated module of
+TopoStats i.e. `process`.
```

Comment:
```suggestion
TopoStats e.g. `process`.
```

...as it's just an [example](https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference) and in due course the other modules will be available.